### PR TITLE
Update DatoCMS

### PR DESCRIPTION
**Pull request by stefanoverna**
Hi! Just wanted to correct some checkbox for DatoCMS! I'll add a link to our docs where available!

### CODE REVIEW

File: datocms.json [hunk 1]
```diff
@@ -97,15 +97,15 @@
       },
       "feature_stemming": {
         "name": "Stemming",
-        "value": "No"
+        "value": "Yes"
```
Comment (stefanoverna) [inline datocms.json:100]: https://www.datocms.com/docs/site-search#how-it-works

File: datocms.json [hunk 2]
```diff
@@ -146,7 +146,7 @@
     },
     "assetFingerprinting": {
       "name": "Asset fingerprinting",
-      "value": "No"
+      "value": "Yes"
```
Comment (stefanoverna) [inline datocms.json:149]: Every asset uploaded is fingerprinted by default!

File: datocms.json [hunk 3]
```diff
@@ -199,7 +199,7 @@
       },
       "react": {
         "name": "React",
-        "value": "No"
+        "value": "Yes"
```
Comment (stefanoverna) [inline datocms.json:202]: https://github.com/datocms/react-datocms

File: datocms.json [hunk 4]
```diff
@@ -249,7 +249,7 @@
     },
     "oauth2Support": {
       "name": "OAuth 2.0 Support",
-      "value": "No"
+      "value": "Yes"
```
Comment (stefanoverna) [inline datocms.json:252]: https://oauth.datocms.com/session/new

File: datocms.json [hunk 5]
```diff
@@ -322,7 +322,7 @@
       },
       "migration": {
         "name": "Migration",
-        "value": "No"
+        "value": "Yes"
```
Comment (stefanoverna) [inline datocms.json:325]: https://www.datocms.com/docs/scripting-migrations/scripting-migrations-with-the-datocms-cli

File: datocms.json [hunk 6]
```diff
@@ -310,7 +310,7 @@
       },
       "branches": {
         "name": "Branches",
-        "value": "No"
+        "value": "Yes"
```
Comment (stefanoverna) [inline datocms.json:313]: https://www.datocms.com/docs/scripting-migrations/introduction#whats-an-environment

**Comment by mad-gav** [comment]
Can we update this soon please?

**Comment by mad-gav** [comment]
Can we not have this approved? this resource needs to be up to date as ppl use it to base decisions on

**Comment by GreatState** [comment]
How long does it take to get this merged?

**Comment by bernhardriegler** [comment]
thank you for the PR and providing all the links to the docs!

**Pull request merged**